### ISSUE TITLE
pre-webmention setup

### ISSUE THREAD

**Issue by kevinmarks** [andybelldesign/hylia#29]
Hi Andy, you said you wanted help with webmentions - did you mean sending or receiving?

If you're going to send them, adding microformats-2 to the templates will make them work better - [indiewebify.me](http://indiewebify.me) will walk you through that.
Then you can use a tool to send webmentions after the site updates - eg https://telegraph.p3k.io/ or https://webmention.app/ or https://mention.tech/

To receive them, if you don't want to run your own code you can use a tool like http://webmention.io, http://webmention.herokuapp.com or https://mention.tech/

Come chat about it at https://chat.indieweb.org/dev/

**Comment by andybelldesign** [comment]
I'm very much into <https://webmention.app> because I know Remy has put a lot of good work into it. I think the trick with this feature is going to be similar to the pre-configured CMS—a way to make things easier for folks.

With that in mind, I'm thinking that we provide the following:
- A simple, accessible way to render processed webmentions on a page 
- A way for users to add a link to their webmention at then end of an article (front-end only)
- Make sure the articles contain the right classes / micro formats 

Basically, I think we should follow Max’s lead from this excellent guide: <https://mxb.dev/blog/using-webmentions-on-static-sites/>.
